<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditQuestion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>471</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditAnswer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>471</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditExample">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>471</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>471</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>471</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Example</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>380</y>
     <width>411</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelCounter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxReversable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reversable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNew">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNewOnThis">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New
Based On This</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNewOnThisRev">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New
 Based On
This Reversed</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrev">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
